### ISSUE TITLE
Question about STA

### ISSUE THREAD

**Issue by yinian-lw** [hao-ai-lab/FastVideo#305]
Thank you for the author's work, which has made an outstanding contribution to the acceleration of diffusion video. 

In the STA paper, the authors employed a distillation loss to ensure that the results after incorporating STA remain consistent with the original model, thus requiring only a small number of training steps. 

The ultimate goal is to achieve inference acceleration using STA. I have a question: Can STA be utilized to accelerate training as well? 

For example, based on the Hunyuan pretrained model, using custom data to fine-tune downstream tasks without distillation loss—could this ultimately achieve the same effect as not using STA for acceleration, thereby realizing training acceleration?

**Comment by yiboz2001** [comment]
+1

**Comment by SolitaryThinker** [comment]
We have now release Video Sparse Attention, which is a sparse attention kernel that can be used to accelerate training. Checkout the paper: https://arxiv.org/abs/2505.13389

Blog: https://hao-ai-lab.github.io/blogs/fastvideo_post_training/
